web/shapeworks/public/forms/groom.json

```diff
@@ -306,7 +306,7 @@
                     ],
                     "x-class": "float-right pl-5",
                     "x-display": "custom-conditional",
-                    "x-display-if": "(model) => model.section_2.alignment",
+                    "x-display-if": "(model) => model.section_3.alignment",
```

Comment:
I'm not seeing the alignment method even when alignment is checked.  It looks like it is actually called `alignment_enabled`.